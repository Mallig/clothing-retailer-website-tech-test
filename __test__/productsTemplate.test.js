const pug = require('pug')
const products = require('../public/products')

const compiledFunction = pug.compileFile('views/products.pug')

test('It should render pug correctly', () => {
  expect(compiledFunction(products)).toMatchSnapshot();
});