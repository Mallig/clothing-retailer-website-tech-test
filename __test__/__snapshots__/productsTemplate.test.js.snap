// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should render pug correctly 1`] = `"<d> <p>Welcome to the Shop</p><a href=\\"/basket\\">Basket</a></d><ul id=\\"product-list\\"><li id=\\"product\\"></li><p>Almond Toe Court Shoes, Patent Black</p><p>£99</p><button>-</button><button>+</button><li id=\\"product\\"></li><p>Suede Shoes, Blue</p><p>£42</p><button>-</button><button>+</button><li id=\\"product\\"></li><p>Leather Driver Saddle Loafers, Tan</p><p>£34</p><button>-</button><button>+</button><li id=\\"product\\"></li><p>Flip Flops, Red</p><p>£19</p><button>-</button><button>+</button><li id=\\"product\\"></li><p>Flip Flops, Blue</p><p>£19</p><button>-</button><button>+</button><li id=\\"product\\"></li><p>Gold Button Cardigan, Black</p><p>£167</p><button>-</button><button>+</button><li id=\\"product\\"></li><p>Cotton Shorts, Medium Red</p><p>£30</p><button>-</button><button>+</button><li id=\\"product\\"></li><p>Fine Stripe Short Sleeve Shirt, Grey</p><p>£49.99</p><button>-</button><button>+</button><li id=\\"product\\"></li><p>Fine Stripe Short Sleeve Shirt, Green</p><p>£49.99</p><button>-</button><button>+</button><li id=\\"product\\"></li><p>Sharkskin Waistcoat, Charcoal</p><p>£75</p><button>-</button><button>+</button><li id=\\"product\\"></li><p>Lightweight Patch Pocket Blazer, Deer</p><p>£175.5</p><button>-</button><button>+</button><li id=\\"product\\"></li><p>Bird Print Dress, Black</p><p>£270</p><button>-</button><button>+</button><li id=\\"product\\"></li><p>Mid Twist Cut­Out Dress, Pink</p><p>£540</p><button>-</button><button>+</button></ul>"`;
